doc/source/sdk/v2/zaf_v2.md

```diff
@@ -0,0 +1,197 @@
+## Zendesk App Framework v2 Beta
+The Zendesk App Framework (ZAF) v2 introduces a new way of building Zendesk apps using iframes.
+
+The v2 Framework improves upon v1 by:
+- improving on security by taking advantage of the browsers own cross-domain security features.
+- making it easy to use whatever technologies you're familiar with.
+- providing consistent APIs across all supported Zendesk Products
+
+### How does it work?
+ZAF v2 apps include a manifest.json file and an assets folder. The manifest file includes a slightly modified locations property, which specifies URL paths for each location. These URL paths can be absolute, pointing to an external page, or relative, pointing to a html file inside the assets folder.
+
+e.g.
+```json
+"location": {
+  "zendesk": {
+    "ticket_sidebar":       "assets/iframe.html?location=ticket_sidebar",
+    "new_ticket_sidebar":   "assets/iframe.html?location=new_ticket_sidebar",
+    "user_sidebar":         "assets/user_sidebar.html",
+    "organization_sidebar": "assets/organization_sidebar.html",
+    "nav_bar":              "https://dashboard.myapp.com/nav_bar",
+    "top_bar":              "https://dashboard.myapp.com/top_bar"
+  }
+}
+```
+
+The HTML files referenced on the manifest file will need to include the ZAF SDK v2 library in a script tag. This will allow your iframe to interact with the new ZAF SDK APIs documented below.
```

Comment:
Original:
> The HTML files referenced on the manifest file will need to include ... This will allow your...

Change "on" to "in". Also, use the present tense wherever possible to be more direct:
>  The HTML files referenced in the manifest file need to include ... This allows your iframe....